stdlib/arrayLabels.mli

```diff
@@ -122,7 +142,7 @@ val fill : 'a array -> pos:int -> len:int -> 'a -> unit
 val blit :
   src:'a array -> src_pos:int -> dst:'a array -> dst_pos:int -> len:int ->
     unit
-(** [blit ~src ~src_pos ~dst ~dst_pos ~len] copies [len] elements
+(** [blit src src_pos dst dst_pos len] copies [len] elements
```

Comment:
The docstrings in the `*Labels.mli` files no longer have the `~` for labels. I don't see an easy way to get rid of them using the `unlabel` script, though, if we wanted to keep them in the labeled modules.

Reply:
Then perhaps generating one file by using the other is not the right approach?
Perhaps both files should be handwritten, and we want a test that seds out the labels (including in comments) before `diff`ing the two versions?

Reply:
This is true. However, as noted in the original PR description, I have matched up the names used in the comment description with the labels (before it was srcoff vs ~src_pos).

The new documentation is not incorrect -- you could write that code, so I don't see that it reaches the threshold for separating the files and editing them manually, with all the problems that entails. People using the labeled module documentation can be expected to know how to use labels.

Reply:
Where are we with this? Are people happy with this tradeoff (easier to keep labeled and unlabeled versions in sync <-> lose tildes in the labeled docstrings)? @trefis what do you think?

Reply:
I'm a bit saddened by this, but I recognize this PR brings many improvements and I don't want to block them just for this. That is, as long as other people are fine with it.

PS: « The new documentation is not incorrect -- you could write that code » perhaps, but I definitely consider it back practice to omit labels (and I think many people would agree with me). And it's a bit sad to give the wrong example.

Reply:
I haven't looked at all the files, but a quick look at `string.mli` and `list.mli` only have a `~` in the explanation of how to use their labelled variant. Is it not possible to have the labels in the docstrings and simply strip all `~` characters from the unlabelled mli (possibly after the first doc comment only)? That would be scary in a build system script, but not in one where the resulting diff is itself inspected.